pkg/gameservers/portallocator.go

```diff
@@ -265,7 +271,7 @@ func (pa *PortAllocator) registerExistingGameServerPorts(gameservers []*v1alpha1
 
 	for _, gs := range gameservers {
```

Comment:
should we move this below 
if p.PortPolicy == v1alpha1.Dynamic || p.PortPolicy == v1alpha1.Passthrough {

Reply:
Sorry - not 100% sure I follow?

Reply:
I was thinking that this will be little more efficient :

	for _, p := range gs.Spec.Ports {
		if p.PortPolicy == v1alpha1.Dynamic || p.PortPolicy == v1alpha1.Passthrough {
	             for _, gs := range gameservers {

Reply:
How do we get access to `gs.Spec.Ports`, if we are not within the  `for _, gs := range gameservers` section?

Reply:
My bad, I didn't realize that (without IDE) that it is looping over the gs.Spec.Ports.